pkgs/development/ocaml-modules/dbf/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, buildDunePackage, fetchFromGitHub, ocaml, ppx_cstruct, rresult
+, cstruct-unix, core_kernel }:
+
+buildDunePackage rec {
+  pname = "dbf";
+  version = "0.1.1";
+
+  minimumOCamlVersion = "4.02";
```

Comment:
```suggestion
  minimalOCamlVersion = "4.02";
```

Reply:
The nixpkgs manual indeed uses "minimal" but I also see "minimum" in many OCaml packages. How do these differ ? buildDunePackage seems to use either indiscriminately.

Reply:
“minimum” works but 1. it is a grammatical error and 2. it changes the derivation hash (which is wrong at it is only meta-data). Only the correct spelling is documented.